frontend/src/features/env/EnvService.ts

```diff
@@ -82,28 +59,42 @@ export const submitSwitchEnvFormFeedback = async ({
   formInputs,
   feedbackForm,
 }: {
-  formInputs: SwitchEnvFeedbackFormBodyDto
-  feedbackForm: PublicFormViewDto | undefined
+  formInputs: AdminFeedbackFormDto | PublicFeedbackFormDto
+  feedbackForm?: PublicFormViewDto
 }): Promise<SuccessMessageDto | ErrorDto> => {
-  if (!feedbackForm) return new Error('feedback form not provided')
+  if (!feedbackForm) return new Error('Feedback form not provided')
+
+  const isAdmin = Object.keys(formInputs).includes('rating')
+  const formFields = ['url', 'feedback', 'email', 'rumSessionId']
+
+  if (isAdmin) formFields.push('rating')
+
   const formData = createSwitchFeedbackSubmissionFormData(
+    formFields,
     formInputs,
     feedbackForm,
   )
+
   return ApiService.post<SuccessMessageDto>(
-    `${PUBLIC_FORMS_ENDPOINT}/submissions/email/switchenvfeedback?captchaResponse=null`,
+    `${PUBLIC_FORMS_ENDPOINT}/submissions/email/switchenvfeedback?captchaResponse=null&view=${
```

Comment:
just noticed this API call haha, should we do it in a more idiomatic `axios` way?

```ts
return ApiService.post(
  `${PUBLIC_FORMS_ENDPOINT}/submissions/email/switchenvfeedback`,
  null, // no query body
  { 
    params: {
      captchaResponse: null, // do we even need this? I think it defaults to null on the server validator.
      view: isAdmin ? 'admin' : 'public'
    }
  }
}
  
```

Reply:
captchaResponse is required by `validateCaptchaParams` haha